Jenkinsfile

```
@@ -10,9 +10,24 @@ pipeline {
                 sh 'cp /usr/local/etc/roms/mm.us.rev1.z64 baserom.mm.us.rev1.z64'
             }
         }
+        stage('Setup') {
+            steps {
+                sh 'make -j setup 2> >(tee tools/warnings_count/warnings_setup_new.txt)'
```

I guess you can change to this?
```suggestion
                sh 'bash -c "make -j setup 2> >(tee tools/warnings_count/warnings_setup_new.txt)"'
```